sroka/api/google_drive/google_drive_api.py

```diff
@@ -134,10 +149,11 @@ def google_drive_sheets_write(data, spreadsheet_id, sheet_range='Sheet1!A1',
             body=body, valueInputOption='RAW').execute()
     except HttpError as err:
         if err.resp.status == 400:
-            print("HTTP error occurred. Please verify if sheet_range is correctly set.")
+            print("HTTP error occurred. Please verify if sheet_range is correctly set. Error:")
+            print(err)
```

Comment:
We don't any return nor here nor 3 lines later, so in case of any errors we will get both messages: one about the error and one about successful upload which didn't happen

Reply:
my mistake, thank you!